### ISSUE TITLE
test resnet50 failed?

### ISSUE THREAD

**Issue by zkh2016** [pytorch/benchmark#1354]
Hi，I encountered a problem when executing the following test command :
```
 python run.py resnet50 -d cuda -t train --bs 32 --torchdynamo inductor
```
The result is:
```
...
[2022-12-28 06:50:30,069] torch._inductor.ir: [WARNING] Using FallbackKernel: torch.ops.aten.addcmul_.default
[2022-12-28 06:50:30,087] torch._inductor.ir: [WARNING] Using FallbackKernel: torch.ops.aten.addcmul_.default
[2022-12-28 06:50:42,387] torch._inductor.compile_fx: [WARNING] skipping cudagraphs due to multiple devices
grad of param conv1.weight after running with dynamo doesn't have gradient matching with eager mode
Running train method from resnet50 on cuda in eager mode with input batch size 32.
GPU Time:             38.122 milliseconds
CPU Total Wall Time:  38.148 milliseconds
Correctness:                        False
```
It shows that the correctness is false. Is it abnormal?

env: [pytorch2.0](https://pytorch.org/get-started/pytorch-2.0/#getting-started) , A100, cuda11.7

**Comment by xuzhao9** [comment]
I can reproduce it on the latest PyTorch nightly and A100:

```
Running train method from resnet50 on cuda in eager mode with input batch size 32.
GPU Time:             37.284 milliseconds
CPU Total Wall Time:  37.334 milliseconds
Correctness:                        False
```
version: torch-2.0.0.dev20221222+cu116

@desertfire Can you help look into this and see if you can reproduce?

**Comment by desertfire** [comment]
I can reproduce it locally, but running resnet50 through the script in PyTorch core does pass (`python benchmarks/dynamo/torchbench.py --training --accuracy --device cuda --inductor  --only resnet50`). So I suspect there is something inconsistent between two scripts.

**Comment by xuzhao9** [comment]
Fixed by https://github.com/pytorch/benchmark/pull/1700
